src/pages/v2/p/components/V2Dashboard.tsx

```diff
@@ -233,11 +243,11 @@ export default function V2Dashboard({ projectId }: { projectId: number }) {
     },
 
     veNft: {
-      name: undefined,
+      name: veNftProjectName,
       lockDurationOptions: lockDurationOptions,
-      baseImagesHash: undefined,
-      resolverAddress: undefined,
-      variants: undefined,
+      baseImagesHash: baseImagesHash,
+      resolverAddress: resolverAddress,
+      variants: variants,
       userTokens: undefined,
     },
```

Comment:
**question** I should have asked this in earlier PR, but:

Is it essential/required that we have veNft data in our top-level context?

This code executes on every project page load. But, I don't think we _need_ the veNft data until the user opens the drawer. Is that correct?

If so, wydt about only loading the related veNft data when the drawer is opened?

Where possible we want to limit the number network requests performed upfront.

Reply:
Agreed, I put it at the same scope as the NftProjectRewards data but that information is actually used in the main project page as opposed to the veNft stuff.

Will leave to a prioritized optimization issue for now.

Reply:
Tracked in #1529